src/common.ts

```diff
@@ -57,3 +60,8 @@ export const isValidTerraAddress = (address: string): boolean => {
 export const isValidTerraTx = (tx: string): boolean => {
   return typeof tx === 'string' && tx.length === 64
 }
+
+export const getRSKChainID = (chain: string, network: string) => {
```

Comment:
why does "chain" need to be passed in if it's rsk specific function?

If you want to go the route of supporting any chain, you can put `getChainId` on the `Chain` interface and install `@liquality/ethereum-networks` to retrieve the chainId for any network
but I think that's overkill for now